### DISCUSSION TITLE
My repository is gone!

### DISCUSSION THREAD

**Discussion by ViridianTelamon** [community/community#39590] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So, I had a repository called A Cool Terminal Showcase and it was written in Python.  Basically it was a terminal showcase of character movement and pushing around an object.  I haven't really checked my repositories for a while because usually what I do is I post the project when is done.  I posted this repository in late October to early November I think.  I'm really hope someone can help me understand what happened because I know FOR SURE I didn't delete it, I even looked at it a few days after I posted it.

**Comment by VladislavDev** [comment]
Perhaps you just looked at the Overview tab, where only the most popular repositories are shown. On the repositories tab of your account, I see this repository. [Here it is](https://github.com/ViridianTelamon/A-Cool-Terminal-Showcase)

**Reply by ViridianTelamon** [reply]
Yeah, this is a project I made and finished a few months ago.

**Reply by ViridianTelamon** [reply]
I remade it now, so it's fine.
